sycl/source/CMakeLists.txt

```diff
@@ -4,9 +4,17 @@
 #cmake_policy(SET CMP0057 NEW)
 #include(AddLLVM)
 
+include_directories(${LLVM_EXTERNAL_XPTI_SOURCE_DIR}/include)
+
 function(add_sycl_rt_library LIB_NAME)
+  # Add an optional argument so we can get the library name to
+  # link with for Windows Debug version
+  cmake_parse_arguments(ARG "" "XPTI_LIB" "" ${ARGN})
+  if (NOT DEFINED ARG_XPTI_LIB)
+    set(ARG_XPTI_LIB xpti)
+  endif()
```

Comment:
Instead of doing that I suggest that you pass xpti lib for linking to add_sycl_rt_library (line 133)

Reply:
Done!